requirements/edx/github.txt

```diff
@@ -47,7 +47,6 @@
 -e git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline
 git+https://github.com/edx/django-wiki.git@v0.0.5#egg=django-wiki==0.0.5
 git+https://github.com/edx/django-openid-auth.git@0.8#egg=django-openid-auth==0.8
--e git+https://github.com/edx/django-rest-framework-oauth.git@f0b503fda8c254a38f97fef802ded4f5fe367f7a#egg=djangorestframework-oauth==1.0.1
```

Comment:
Thanks for the tip. Does this uninstall need to happen if the egg name is unchanged (i.e., if we decide to go back to using the unforked `djangorestframework-oauth`)? It looks like it might, since the package was previously installed as editable.

Also, it looks like this uninstall caused problems when creating new Jenkins workers. Do you know if those issues were resolved?